src/main/java/io/confluent/connect/jdbc/util/CachedConnectionProvider.java

```diff
@@ -57,8 +58,10 @@ public synchronized Connection getConnection() {
         newConnection();
       }
     } catch (SQLException sqle) {
+      log.debug("Could not establish connection with database.");
```

Comment:
log level should be debug or error?

Reply:
Kept this as debug since this method is called multiple times. Could lead to frequent logging at error level.